### PR TITLE
DOC: README.md wrong minimum versions

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,8 +165,8 @@ pip install pandas
 
 ## Dependencies
 - [NumPy](https://www.numpy.org): 1.13.3 or higher
-- [python-dateutil](https://labix.org/python-dateutil): 2.5.0 or higher
-- [pytz](https://pythonhosted.org/pytz): 2015.4 or higher
+- [python-dateutil](https://labix.org/python-dateutil): 2.6.1 or higher
```
Comment (mroeschke) [inline README.md]: Instead of having to manage this in 2 places, I think we should just link to our docs.

Comment (MomIsBestFriend) [reply README.md]: You mean, completely remove the "Dependencies" section from README.md, and put something like (sorry in advanced for my English):


### To see the minimum required versions:
click [here](https://pandas.pydata.org/pandas-docs/stable/install.html#dependencies)


______

Or there is an actual option to link the version from the web directly to the README.md?

**Comment by MomIsBestFriend** [comment]
How did it fail the tests?

File: README.md (hunk 1 above)
Comment (mroeschke) [reply README.md]: Maybe just remove the version numbers here and clarify the sentence below to say:

`See the full installation instructions for minimum supported versions of required, recommended and optional dependencies.`

Comment (MomIsBestFriend) [reply README.md]: Ty so much @mroeschke , you saved me a huge headache of how to format the message.

Really appreciate it.

**Comment by jbrockmendel** [comment]
test failures are unrelated.  LGTM

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @MomIsBestFriend !

**Pull request merged**